src/camTempMonitor.h

```diff
@@ -27,14 +27,22 @@ Q_DECLARE_METATYPE(std::vector<double>)
     thus a recEventTracker instance also must exist
 */
 
+/** The temperature that we deem as default value when cannot be measured, and the minimum that can be measured */
+static const double MINIMUM_DEVICE_TEMPERATURE = -1.0;
```

Comment:
This should not be left as a global variable

Reply:
Made it inside the class, but could not access it elsewhere, left it like this with a comment